<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="mainFrame" >
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item>
       <widget class="QtChatHistoryWidget" name="chatHistory" >
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtWengoStyleBar" name="editActionBar" >
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="editFrame" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>5</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>16</width>
          <height>56</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>16777215</width>
          <height>56</height>
         </size>
        </property>
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_3" >
           <property name="frameShape" >
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QtChatEditWidget" name="chatEdit" >
              <property name="frameShape" >
               <enum>QFrame::NoFrame</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="sendButton" >
           <property name="minimumSize" >
            <size>
             <width>56</width>
             <height>56</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
           <property name="icon" >
            <iconset resource="../../../../resources/wengophone.qrc" >:/pics/chat/chat_send.png</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QtChatAvatarFrame" name="avatarFrame" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>13</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="avatarFrameButton" >
       <property name="minimumSize" >
        <size>
         <width>14</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>14</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../../../../resources/wengophone.qrc" >:/pics/chat/hide_avatar_frame.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>14</width>
         <height>31</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>509</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>QtChatAvatarFrame</class>
   <extends>QFrame</extends>
   <header>chat/QtChatAvatarFrame.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>QtWengoStyleBar</class>
   <extends>QFrame</extends>
   <header>chat/QtWengoStyleBar.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>QtChatEditWidget</class>
   <extends>QTextEdit</extends>
   <header>chat/QtChatEditWidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>QtChatHistoryWidget</class>
   <extends>QTextBrowser</extends>
   <header>chat/QtChatHistoryWidget.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
